developer/ckcp/README.md

```diff
@@ -26,6 +26,7 @@ Before you execute the script, you need:
 4. to install [yq](https://mikefarah.gitbook.io/yq/#install)
 6. to install [kubectl kcp plugin](https://github.com/kcp-dev/kcp/blob/main/docs/kubectl-kcp-plugin.md)
    Note: ckcp uses the official kcp image in order to run kcp in a pod (latest released versions). It is advisable to use the same version for the kcp plugin as the kcp core API (especially as KCP API is evolving quickly). The current version can be found in [DEPENDENCIES.md](../DEPENDENCIES.md). Make sure to checkout the branch listed in the doc before installing the plugin.
+7. to install [podman](https://github.com/containers/podman) or [docker](https://www.docker.com/). If you have both you control which is used by setting the `CONTAINER_ENGINE` environment variable. If using podman, it is recommended to run `export ALLOW_ROOTLESS=true` to run the containers with your user instead of `root` (note that this is unnecessary if you've already set `CONTAINER_ENGINE` to `podman`).
```

Comment:
> (note that this is unnecessary if you've already set `CONTAINER_ENGINE` to `podman`).
This sentence is a bit confusing. Why would setting export ALLOW_ROOTLESS=true be unnecessary if container_engine is set to podman?

Reply:
ALLOW_ROOTLESS dropped from the scripts and doc.